src/VirtoCommerce.WebHooksModule.Data/Services/WebHookManager.cs

```diff
@@ -107,12 +110,18 @@ protected virtual async Task HandleEvent(DomainEvent domainEvent, CancellationTo
 
             if (webHookSearchResult.TotalCount > 0)
             {
-                var request = new WebhookRequest { EventId = eventId, EventObject = domainEvent, WebHooks = webHookSearchResult.Results };
+                var eventObject = domainEvent.GetChangedEntriesWithInterface<IEntity>().Select(x => new { x.Id }).FirstOrDefault();
+                var request = new WebhookRequest
```

Comment:
Are we gping to send only the first object Id - is it acceptable by the requirements?
Also couldn't there be a null reference exception on selecting x => x.Id (e.g. Null object in GenericChangedEntry event, like for Add operation)

Reply:
we decide to send only Id of object event.
@OlegoO confirmed pls

Reply:
@kostyrin I confirmed that If we need to improve performance, we can minimize event with ObjectId and ObjectType

Reply:
ok will add ObjectType